### DISCUSSION TITLE
Can I join your team?

### DISCUSSION THREAD

**Discussion by Yara0202** [ROM-builders/temporary#16906] [Q&A]
My Github Profile : https://github.com/Hirokixd
My Telegram : https://t.me/Hirokixd28

My Successful Build Xdroid:
https://cirrus-ci.com/build/5325343371296768

DT : https://github.com/Hirokixd/device_asus_X01AD/tree/xdroid-twelve

VT :
https://github.com/Hirokixd/vendor_asus_X01AD/tree/A-12.1

KT :
https://github.com/Hirokixd/kernel_asus_X01AD/tree/lineage-19.1

**Comment by Apon77** [comment]
please follow #learn instructions in telegram group
